src/test/java/org/numenta/nupic/network/LayerTest.java

```diff
@@ -307,7 +311,7 @@ public void testHalt() {
         Parameters p = NetworkTestHarness.getParameters().copy();
         p = p.union(NetworkTestHarness.getHotGymTestEncoderParams());
         p.set(KEY.RANDOM, new MersenneTwister(42));
-        p.set(KEY.AUTO_CLASSIFY, Boolean.TRUE);
+        p.set(KEY.AUTO_CLASSIFY, false);
```

Comment:
If it is not stated at all, I believe it will be regarded as false, as that is the default setting for the `KEY.AUTO_CLASSIFY` key.
 
Yes, they should remain the same. What I should have done with these tests (and what I _did_ do with all the others) was add a mapping of the `consumption` field to a classifier type to the `KEY.INFERRED_FIELDS` parameter.